charts/studio/values.yaml

```diff
@@ -1313,38 +1303,6 @@ vector-aggregator:
           key: token
 
   extraObjects:
-    - apiVersion: rbac.authorization.k8s.io/v1
-      kind: ClusterRole
-      metadata:
-        name: studio-vector-aggregator-extended
-        annotations:
-          helm.sh/hook: pre-install, pre-upgrade
-          helm.sh/hook-delete-policy: hook-failed, before-hook-creation
-
-      rules:
-        - apiGroups:
-            - ""
-          resources:
-            - events
-          verbs:
-            - get
-            - list
-
-    - apiVersion: rbac.authorization.k8s.io/v1
-      kind: ClusterRoleBinding
-      metadata:
-        name: studio-vector-aggregator-extended
-        annotations:
-          helm.sh/hook: pre-install, pre-upgrade
-          helm.sh/hook-delete-policy: hook-failed, before-hook-creation
-      subjects:
-        - kind: ServiceAccount
-          name: studio-vector-aggregator
-          namespace: default  # FIXME: should be configurable
```

Comment:
@shcheklein, it manly fixes this (multi-namespace support, useful for https://github.com/iterative/studio/pull/12126) and also cleans up the bloated `extraObjects` configuration needed before.

Reply:
good, just so that you are aware though - our recent experience, vector overall is very unstable - it drop the logs (or may be k8s itself drop them I don't know - ultimately we just see a small fraction of what should be there). Not sure if this one fixes it or not ....